Sources/iron/Scene.hx

```diff
@@ -193,10 +193,14 @@ class Scene {
 		if (!framePassed) return;
 		framePassed = false;
 
+		// Defer unloading the world shader until the new world shader is loaded
+		// to prevent errors due to a missing world shader inbetween
+		var removeWorldShader: Null<String> = null;
```

Comment:
Why is the `Null<>` boxing needed?

Reply:
It isn't really needed but I like to use it to make it more verbose whether a variable can hold null values or not (even though in this case it's really not needed). There are a lot of cases where it's easier to just hover over the variable in the IDE instead of searching through all uses before knowing if it can hold null or not. But feel free to remove it as it really is quite useless here.

Are there some performance implications by using `Null<>` for types that are already nullable?

Reply:
Might be nice to eventually enforce that globally so it is consistent and can be relied upon. Right now (as we know) the variables not marked as `Null<>` can still be null all over the code base.

> Are there some performance implications by using Null<> for types that are already nullable?

Not for JS, not sure about others.